.env-sample

```diff
@@ -0,0 +1,11 @@
+TAB_SEARCH_URL=
+TAB_ARTIST_SONGS_INDEX=
+TAB_BAND_INDEX=
+AUTH0_CLIENT_ID=
+AUTH0_CLIENT_SECRET=
+AUTH0_DOMAIN=
+DATABASE_URL=
+DEEP_LINK=
+DEEP_LINK_SCHEME=
+SPOTIPY_CLIENT_ID=
+SPOTIPY_CLIENT_SECRET=
```

Comment:
I figured it's a good idea to make needing the `.env` file known. @samdatkins , are there any of these that would be after to be populated? For example, is it safe to have `TAB_SEARCH_URL` ?

Reply:
Variable names are a good idea, let's keep it at that and we can just distribute test values by hand